about.html

```diff
@@ -0,0 +1,172 @@
+            <li>
+              <a href="#"><i class="uil uil-facebook-f"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="uil uil-twitter"></i></a>
+            </li>
+            <li>
+              <a href="#">English</a>
+            </li>
+            <li>
+              <a href="#">My Page</a>
+            </li>
+            <li>
+              <a href="#">Logout</a>
+            </li>
+          </ul>
+        </div>
+
+        <div class="bottom">
+          <div class="container">
+            <a href="index.html" class="nav-logo">ILC 2023</a>
+            <ul class="nav-menu">
+              <li class="nav-item">
+                <a href="index.html" class="nav-link">Home</a>
+              </li>
+              <li class="nav-item">
+                <a href="about.html" class="nav-link">About</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Program</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Join</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link">News</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link campaign">CC Campaign</a>
+              </li>
+            </ul>
+            <div class="hamburger">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <!-- HEADER ENDS -->
+
+    <!-- MAIN STARTS -->
+
+    <main>
+      <section class="main-page about-page container">
+        <p class="greet about-hide">"Lead the world through innovations."</p>
+        <h1 class="about-header">
+          Connecting Minds, <br />
+          Empowering Change 2023
+        </h1>
+        <p class="description about-description">
+          "Welcome to the Innovation and Leadership Conference, a premier event
+          dedicated to fostering creativity, driving innovation, and promoting
+          effective leadership in today's dynamic and fast-paced business
+          environment."
+        </p>
+        <p class="about-location">
+          Please contact us via email for any further question and
+          clarifications about the ILC Summit 2023
+        </p>
+        <a class="email" href="#">info@ilcsummit2023.org</a>
+      </section>
+    </main>
+    <!-- MAIN ENDS -->
+
+    <!-- SUB-SECTION STARTS -->
+    <section class="sub-section">
+      <div class="container">
+        <h2>ILC Summit 2023 Logo</h2>
+        <hr />
+        <p>
+          The Logo of the ILC summit 2023 will be decided through the annual ILC
+          logo competition on May 8, 2023.
+        </p>
```

Comment:
- [ ] I believe you just merged two sections into one which might not necessarily be the best approach here. How about breaking this up, and then include the currently missing logo that you were introducing in your paragraph just as it is in the design? :hugs: 

_Check this out:_
|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/233463246-d23bcceb-71d4-4029-84c1-ab1df58166a0.png)|![image](https://user-images.githubusercontent.com/99715640/233463428-1e6bcc3c-d3a1-40af-9cba-e48474a16e20.png)|

- Looks like title here also has a divider